### PR TITLE
Update html_to_widgets.dart + css

### PR DESCRIPTION
**Pull request by Ammarmahmoud1995**
I added support to Css( font size , direction , color, line break ) this mainly help me with html enhanced editor

### CODE REVIEW

File: lib/src/html_to_widgets.dart [hunk 1]
```diff
@@ -330,6 +333,14 @@ class WidgetsHTMLDecoder {
     final children = element.nodes.toList();
     final childNodes = <Widget>[];
     for (final child in children) {
+
```
Comment (alihassan143) [inline lib/src/html_to_widgets.dart:336]: kindly create separate function for css attributes

File: lib/src/html_to_widgets.dart [hunk 2]
```diff
@@ -330,6 +333,14 @@ class WidgetsHTMLDecoder {
     final children = element.nodes.toList();
     final childNodes = <Widget>[];
     for (final child in children) {
+  
+
+            print(child.parent!.outerHtml);
```
Comment (alihassan143) [inline lib/src/html_to_widgets.dart:338]: remove print statements and format the code

File: lib/src/html_to_widgets.dart [hunk 3]
```diff
@@ -342,15 +353,102 @@ class WidgetsHTMLDecoder {
...
+   alignmentfortext = Alignment.centerRight ; 
+}
+
+if(child.text! =="."){
+  selectedcolor = PdfColors.white ;
+}
+
+
+
+           print("hiiiooooo") ;
```
Comment (alihassan143) [inline lib/src/html_to_widgets.dart:444]: also remove print here

File: lib/src/html_to_widgets.dart [hunk 4]
```diff
@@ -342,15 +353,102 @@ class WidgetsHTMLDecoder {
...
-            delta.add(Text(child.text, style: attributes));
+            delta.add(  Align(
+            alignment:alignmentfortext, child:Text(child.text , style: attributes.copyWith(fontSize: fontsizee ,color: selectedcolor))));
           }
         }
       } else {
-        delta.add(Text(child.text ?? "",
-            style: TextStyle(font: font, fontFallback: fontFallback)
-              ..merge(customStyles.paragraphStyle)));
+Map<String,double> fontss = {
```
Comment (alihassan143) [inline lib/src/html_to_widgets.dart:404]: create separate function

**Review by alihassan143 — CHANGES_REQUESTED** [review]
kindly apply some changes remove print and make seperate functions

**Comment by alihassan143** [comment]
@Ammarmahmoud1995  also create issue for that

**Comment by Ammarmahmoud1995** [comment]
I commit the changes  , hope I didn't miss something  
With all regards

**Comment by alihassan143** [comment]
@Ammarmahmoud1995  i did some changes kindly add some test related to your function please

**Comment by Ammarmahmoud1995** [comment]
So I did some unit testing in flutter test covered 6 cases ,,, I used records in expect function ...  am not sure if this is exactly what you are looking for .. hope it is useful  
````
List input = ['''<div dir="rtl" style="text-align: left;"><font color="#c437c4" size="5">Ammar mahmoud 1</font></div>''',
'''<div dir="rtl" style="text-align: left;"><font color="#c437c4" size="5"><br></font></div>''',
'''<div dir="rtl" style="text-align: center;"><font size="6">2 عمار محمود&nbsp;</font></div>''',
'''<div dir="rtl" style="text-align: center;"><font size="6"><br></font></div>''',
'''<div dir="rtl" style="text-align: center;"><font color="#583f3e" size="6">3 عمار Mahmoud</font></div>''',
'''<div dir="rtl" style="text-align: right;">4 السلام وعليكم</div>'''
] ;
List expectedValue = [CssProberties(fontsize: 18.0,alignmentfortext: Alignment.centerLeft,selectedcolor:PdfColor(0.7686274509803922, 0.21568627450980393, 0.7686274509803922, 1)),
CssProberties(fontsize: 18.0,alignmentfortext: Alignment.centerLeft,selectedcolor:PdfColor(0.7686274509803922, 0.21568627450980393, 0.7686274509803922, 1) ),
CssProberties(fontsize: 24.0,alignmentfortext: Alignment.center,selectedcolor:null ),
CssProberties(fontsize: 24.0,alignmentfortext: Alignment.center,selectedcolor:null ),
CssProberties(fontsize: 24.0,alignmentfortext: Alignment.center,selectedcolor:PdfColor(0.34509803921568627, 0.24705882352941178, 0.24313725490196078, 1), ),
CssProberties(fontsize: null,alignmentfortext: Alignment.centerRight,selectedcolor:null ),
]  ;
void main() {    
   for(int i = 0 ; i<input.length ; i ++) {
   test("test _handleCss",(){  
      var expected = expectedValue[i];     
  (double?,Alignment?,PdfColor?)  expectedrecord = (expected.fontsize,expected.alignmentfortext,expected.selectedcolor)  ; 
      var actual = _handleCss(input[i]);   
 (double?,Alignment?,PdfColor?)  actualRecord = (actual.fontsize,actual.alignmentfortext,actual.selectedcolor)  ; 
      expect(actualRecord,expectedrecord); 
   }); }
}
```

**Comment by Ammarmahmoud1995** [comment]
Results where
 6✓ test _handleCss
Multiple tests named "test _handleCss" ran but only one was expected.
You may have multiple tests with the same name.
Exited

**Comment by alihassan143** [comment]
@Ammarmahmoud1995  kindly write inside test folder so ci can check it

**Comment by alihassan143** [comment]
@Ammarmahmoud1995 also remove the conflicts

**Pull request closed without merging**